src/components/data-processing/CellSizeDistribution/CalculationConfig.jsx

```diff
@@ -1,97 +1,19 @@
-import React, { useState } from 'react';
-import { useSelector, useDispatch } from 'react-redux';
+import React from 'react';
 import PropTypes from 'prop-types';
 import {
-  Space,
   InputNumber,
   Form,
-  Button,
-  Alert,
-  Radio,
 } from 'antd';
 
-import _ from 'lodash';
-
 import BandwidthOrBinstep from '../ReadAlignment/PlotStyleMisc';
 
-import { updateProcessingSettings } from '../../../redux/actions/experimentSettings';
-
-const CalculationConfig = (props) => {
+const CellSizeDistributionConfig = (props) => {
```

Comment:
very good idea to refactor this away into a parent config component `CalculationConfigContainer`.

Reply:
thanks :-)